conda/install.py

```diff
@@ -287,9 +288,25 @@ def create_meta(prefix, dist, info_dir, extra_info):
     meta_dir = join(prefix, 'conda-meta')
     if not isdir(meta_dir):
         os.makedirs(meta_dir)
-    with open(join(meta_dir, dist + '.json'), 'w') as fo:
-        json.dump(meta, fo, indent=2, sort_keys=True)
-
+    shutil.copytree(info_dir, join(meta_dir, dist))
+
+    if 'files' in extra_info:
+        with open(join(meta_dir, dist, 'files'), 'wb') as fp:
```

Comment:
This should probably go in an env specific directory so it doesn't overwrite anything that's already there.